Modules/@babylonjs/react-native/ios/BabylonNative.cpp

```diff
@@ -57,8 +56,7 @@ namespace Babylon
         // React Native's implementation, but rather adds a second one scoped to Babylon and used by WebRequest.ts.
         Polyfills::XMLHttpRequest::Initialize(m_impl->env);
 
-        Plugins::NativeWindow::Initialize(m_impl->env, windowPtr, width, height);
-        Plugins::NativeEngine::Initialize(m_impl->env);
+        Plugins::NativeEngine::Initialize(m_impl->env, true);
```

Comment:
nit: `true` is the default value for that argument; you can specify it explicitly, but you don't need to.

Reply:
Ok, I think I'd rather be explicit with this arg.